2013-10-31T23:53:58.856458+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
2013-10-31T23:53:58.856928+00:00 app[web.1]:     {% extends "layout.html" -%}
2013-10-31T23:53:58.856928+00:00 app[web.1]:     urls = bundle.urls(env=env)
2013-10-31T23:53:58.856928+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 681, in urls
2013-10-31T23:53:58.856928+00:00 app[web.1]:     urls.extend(bundle._urls(env, extra_filters, *args, **kwargs))
2013-10-31T23:53:58.857081+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 643, in _urls
2013-10-31T23:53:58.857081+00:00 app[web.1]:     *args, **kwargs)
2013-10-31T23:53:58.857081+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 498, in _build
2013-10-31T23:53:58.857081+00:00 app[web.1]:     kwargs=item_data)
2013-10-31T23:53:58.857081+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 269, in apply
2013-10-31T23:53:58.857081+00:00 app[web.1]:     return self._wrap_cache(key, func)
2013-10-31T23:53:58.857081+00:00 app[web.1]:     content = func().getvalue()
2013-10-31T23:53:58.856458+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
2013-10-31T23:53:58.856299+00:00 app[web.1]:     rv = self.dispatch_request()
2013-10-31T23:53:58.856458+00:00 app[web.1]:   File "/app/app.py", line 45, in index
2013-10-31T23:53:58.856458+00:00 app[web.1]:     return self.environment.handle_exception(exc_info, True)
2013-10-31T23:53:58.856928+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-10-31T23:53:58.856928+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/ext/jinja2.py", line 181, in _render_assets
2013-10-31T23:53:58.856299+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-10-31T23:53:58.857081+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 423, in _merge_and_apply
2013-10-31T23:53:58.857081+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 216, in _wrap_cache
2013-10-31T23:53:58.856458+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
2013-10-31T23:53:58.856928+00:00 app[web.1]:   File "/app/templates/index.html", line 1, in top-level template code
2013-10-31T23:53:58.856928+00:00 app[web.1]:   File "/app/templates/layout.html", line 5, in top-level template code
2013-10-31T23:53:58.856928+00:00 app[web.1]:     {% assets "less" -%}
2013-10-31T23:53:58.857081+00:00 app[web.1]:     force, disable_cache=disable_cache, extra_filters=extra_filters)
2013-10-31T23:53:58.857447+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 249, in func
2013-10-31T23:53:58.857447+00:00 app[web.1]:     getattr(filter, type)(data, out, **kwargs_final)
2013-10-31T23:53:58.857447+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/filter/__init__.py", line 481, in subprocess
2013-10-31T23:53:58.857447+00:00 app[web.1]:     errread, errwrite)
2013-10-31T23:53:58.857447+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/subprocess.py", line 1249, in _execute_child
2013-10-31T23:53:58.857447+00:00 app[web.1]:     raise child_exception
2013-10-31T23:53:58.857447+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/filter/less.py", line 83, in input
2013-10-31T23:53:58.857447+00:00 app[web.1]:     self.subprocess([self.less or 'lessc', '-'], out, in_)
2013-10-31T23:53:58.857447+00:00 app[web.1]:     stderr=subprocess.PIPE)
2013-10-31T23:53:58.857619+00:00 app[web.1]: OSError: [Errno 2] No such file or directory
2013-10-31T23:53:58.857447+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/subprocess.py", line 679, in __init__
2013-10-31T23:53:58.856937+00:00 heroku[router]: at=info method=GET path=/ host=jonjuaca.herokuapp.com fwd="85.87.132.23" dyno=web.1 connect=1ms service=34ms status=500 bytes=135
2013-10-31T23:53:50.727942+00:00 heroku[router]: at=info method=GET path=/static host=jonjuaca.herokuapp.com fwd="85.87.132.23" dyno=web.1 connect=1ms service=4ms status=404 bytes=233
2013-10-31T23:56:13.728341+00:00 heroku[router]: at=info method=GET path=/tech/ host=jonjuaca.herokuapp.com fwd="85.87.132.23" dyno=web.1 connect=2ms service=29ms status=500 bytes=135
2013-10-31T23:56:13.722541+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 131, in handle_request
2013-10-31T23:56:13.722541+00:00 app[web.1]: Traceback (most recent call last):
2013-10-31T23:56:13.722541+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/werkzeug/contrib/fixers.py", line 144, in __call__
2013-10-31T23:56:13.722541+00:00 app[web.1]: 2013-10-31 23:56:13 [7] [ERROR] Error handling request
2013-10-31T23:56:13.722541+00:00 app[web.1]:     return self.wsgi_app(environ, start_response)
2013-10-31T23:56:13.722541+00:00 app[web.1]:     return self.app(environ, start_response)
2013-10-31T23:56:13.722541+00:00 app[web.1]:     response = self.make_response(self.handle_exception(e))
2013-10-31T23:56:13.722704+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-10-31T23:56:13.722541+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
2013-10-31T23:56:13.722704+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
2013-10-31T23:56:13.722704+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
2013-10-31T23:56:13.722541+00:00 app[web.1]:     respiter = self.wsgi(environ, resp.start_response)
2013-10-31T23:56:13.722704+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-10-31T23:56:13.722704+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
2013-10-31T23:56:13.722847+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/templating.py", line 110, in _render
2013-10-31T23:56:13.722704+00:00 app[web.1]:     rv = self.dispatch_request()
2013-10-31T23:56:13.722704+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
2013-10-31T23:56:13.722847+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
2013-10-31T23:56:13.722847+00:00 app[web.1]:     rv = template.render(context)
2013-10-31T23:56:13.722847+00:00 app[web.1]:     return self.view_functions[rule.endpoint](**req.view_args)
2013-10-31T23:56:13.723209+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 681, in urls
2013-10-31T23:56:13.722847+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/templating.py", line 128, in render_template
2013-10-31T23:56:13.722541+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
2013-10-31T23:56:13.722704+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
2013-10-31T23:56:13.723209+00:00 app[web.1]:     {% assets "less" -%}
2013-10-31T23:56:13.722847+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
2013-10-31T23:56:13.722847+00:00 app[web.1]:     return render_template('tech.html', title='Technologies')
2013-10-31T23:56:13.723209+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/ext/jinja2.py", line 181, in _render_assets
2013-10-31T23:56:13.722847+00:00 app[web.1]:     return self.environment.handle_exception(exc_info, True)
2013-10-31T23:56:13.723391+00:00 app[web.1]:     *args, **kwargs)
2013-10-31T23:56:13.723391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 498, in _build
2013-10-31T23:56:13.722704+00:00 app[web.1]:     response = self.full_dispatch_request()
2013-10-31T23:56:13.722847+00:00 app[web.1]:   File "/app/app.py", line 57, in tech
2013-10-31T23:56:13.723209+00:00 app[web.1]:     reraise(exc_type, exc_value, tb)
2013-10-31T23:56:13.722847+00:00 app[web.1]:     context, ctx.app)
2013-10-31T23:56:13.723209+00:00 app[web.1]:     urls = bundle.urls(env=env)
2013-10-31T23:56:13.723209+00:00 app[web.1]:   File "/app/templates/tech.html", line 1, in top-level template code
2013-10-31T23:56:13.723391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 643, in _urls
2013-10-31T23:56:13.723391+00:00 app[web.1]:     kwargs=item_data)
2013-10-31T23:56:13.723391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/bundle.py", line 423, in _merge_and_apply
2013-10-31T23:56:13.722704+00:00 app[web.1]:     rv = self.handle_user_exception(e)
2013-10-31T23:56:13.723209+00:00 app[web.1]:   File "/app/templates/layout.html", line 5, in top-level template code
2013-10-31T23:56:13.723209+00:00 app[web.1]:     urls.extend(bundle._urls(env, extra_filters, *args, **kwargs))
2013-10-31T23:56:13.723391+00:00 app[web.1]:     force, disable_cache=disable_cache, extra_filters=extra_filters)
2013-10-31T23:56:13.723391+00:00 app[web.1]:     content = func().getvalue()
2013-10-31T23:56:13.723209+00:00 app[web.1]:     {% extends "layout.html" -%}
2013-10-31T23:56:13.723676+00:00 app[web.1]:     errread, errwrite)
2013-10-31T23:56:13.723676+00:00 app[web.1]:     stderr=subprocess.PIPE)
2013-10-31T23:56:13.723391+00:00 app[web.1]:     return self._wrap_cache(key, func)
2013-10-31T23:56:13.723209+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
2013-10-31T23:56:13.723391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 216, in _wrap_cache
2013-10-31T23:56:13.723676+00:00 app[web.1]:     self.subprocess([self.less or 'lessc', '-'], out, in_)
2013-10-31T23:56:13.723391+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 269, in apply
2013-10-31T23:56:13.723676+00:00 app[web.1]:     raise child_exception
2013-10-31T23:56:13.723676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/subprocess.py", line 1249, in _execute_child
2013-10-31T23:56:13.723676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/filter/less.py", line 83, in input
2013-10-31T23:56:13.723676+00:00 app[web.1]:     getattr(filter, type)(data, out, **kwargs_final)
2013-10-31T23:56:13.723676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/subprocess.py", line 679, in __init__
2013-10-31T23:56:13.723676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/merge.py", line 249, in func
2013-10-31T23:56:13.723676+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/webassets/filter/__init__.py", line 481, in subprocess
2013-10-31T23:56:13.723815+00:00 app[web.1]: OSError: [Errno 2] No such file or directory
